contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol

```diff
@@ -9,21 +9,28 @@ import "../interfaces/FunctionsClientInterface.sol";
 import "../../interfaces/TypeAndVersionInterface.sol";
 import "../../interfaces/ERC677ReceiverInterface.sol";
 import "../interfaces/AuthorizedOriginReceiverInterface.sol";
-import "../../ConfirmedOwner.sol";
+import "../ConfirmedOwnerUpgradeable.sol";
 import "../AuthorizedReceiver.sol";
 import "../vendor/openzeppelin-solidity/v.4.8.0/contracts/utils/SafeCast.sol";
-import "../vendor/openzeppelin-solidity/v.4.8.0/contracts/security/Pausable.sol";
-
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+/**
+ * @title Functions Billing Registry contract
+ * @notice Contract that coordinates payment from users to the nodes of the Decentralized Oracle Network (DON).
+ * @dev THIS CONTRACT HAS NOT GONE THROUGH ANY SECURITY REVIEW. DO NOT USE IN PROD.
+ */
 contract FunctionsBillingRegistry is
-  ConfirmedOwner,
-  Pausable,
+  Initializable,
+  ConfirmedOwnerUpgradeable,
+  PausableUpgradeable,
   FunctionsBillingRegistryInterface,
   ERC677ReceiverInterface,
   AuthorizedReceiver
 {
-  LinkTokenInterface public immutable LINK;
-  AggregatorV3Interface public immutable LINK_ETH_FEED;
-  AuthorizedOriginReceiverInterface private immutable ORACLE_WITH_ALLOWLIST;
+  LinkTokenInterface public LINK;
```

Comment:
Have you considered this? https://github.com/smartcontractkit/chainlink/blob/develop/contracts/STYLE.md#visibility

Reply:
This was just lifted from VRFv2, but I will change to maintain our best practices.